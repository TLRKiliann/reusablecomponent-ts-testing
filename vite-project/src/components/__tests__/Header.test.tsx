import {beforeEach, describe, expect, test, it, vi} from 'vitest';
//import {fireEvent, render, screen, waitFor} from '@testing-library/react';
//import { act } from 'react-dom/test-utils';
import {create} from 'react-test-renderer';
import Header from '../Header.tsx'


describe('Sixth snapshot test', () => {
	test('testing Header component', () => {
		let tree = create(<Header />)
		expect(tree.toJSON()).toMatchSnapshot();
	})
})