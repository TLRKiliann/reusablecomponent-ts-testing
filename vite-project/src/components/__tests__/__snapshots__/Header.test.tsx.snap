// Vitest Snapshot v1

exports[`Sixth snapshot test > testing Header component 1`] = `
<div>
  Hello from Header
</div>
`;
